.github/workflows/cypress.yml

```
@@ -54,9 +56,9 @@ jobs:
         uses: cypress-io/github-action@v4
         with:
           install: false
-          start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e API_KEY=test --name forms-api forms-api
+          start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e API_KEY=$SETTINGS__FORMS_API__AUTH_KEY --name forms-api forms-api
```

This might work if you haven't tried it already:
```suggestion
          start: yarn run ci, docker run -p 9292:9292 --network=host -e DATABASE_URL=postgres://rails:password@localhost:5432 -e API_KEY=${{ SETTINGS__FORMS_API__AUTH_KEY }} --name forms-api forms-api
```

---

Worked a treat, thank you @DavidBiddle